rcore-fs-mountfs/src/lib.rs

```diff
@@ -295,6 +295,10 @@ impl INode for MNode {
         self.inode.get_entry(id)
     }
 
+    fn iterate_entry(&self, ctx: &mut DirentWriterContext) -> Result<usize> {
```

Comment:
I have to correct my previous comments. After giving it a second thought, the name of `iterate_entry` uses the wrong plurality. A single call of this function may iterate multiple entries, not one. So to be precise, the name should be `iterate_entries`.